src/database/remove-remote.ts

```diff
@@ -0,0 +1,126 @@
+"use strict";
+
+import { Response } from "request";
+import * as request from "request";
+import responseToError = require("../responseToError");
```

Comment:
```suggestion
import * as responseToError from "../responseToError";
```

Same with all the `require` lines below.